src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
```

Comment:
Suggestion: add TODO comments where we'll need to import and instantiate motor controllers

Question: Will you be doing this yourselves or using the MotorController class?